pkg/platformutil/platformutil.go

```diff
@@ -33,11 +32,7 @@ func ParsePlatforms(all bool, ss string) (platforms.MatchComparer, error) {
 			platforms = append(platforms, plat)
 		}
 	}
-	platformMC, err := NewMatchComparer(all, platforms)
-	if err != nil {
-		return nil, err
-	}
-	return nydusUtils.ExcludeNydusPlatformComparer{MatchComparer: platformMC}, nil
```

Comment:
Seems that no one is using `nydusUtils.ExcludeNydusPlatformComparer` anymore, are we good to remove it entirely?

Reply:
Fixed.